tests/test_manifest.rs

```diff
@@ -12,7 +12,15 @@ fn invalid_manifest() {
             r"Command failed due to unhandled error: Unable to parse Cargo.toml
 
 Caused by: Manifest not valid TOML
-Caused by: failed to parse datetime for key `invalid-section.key`",
+Caused by: TOML parse error at line 6, column 7
+  |
+6 | key = invalid-value
+  |       ^
+Unexpected `i`
+Expected `-`, `+`, `0`, `-`, `+` or `0`
+expected 4 more elements
+expected 2 more elements
+While parsing an Integer",
```

Comment:
Not the end of the world, but that error message looks a bit odd.

Reply:
That's the downside of using a parser combinator library like [combine](https://github.com/Marwes/combine): we can't really control the error messages in every case. So I guess `-`, `+`, `0` is repeated twice here, because integer parser is used in parsing both integers and floats, `expected n more elements` may come from the datetime parser (4 - date, 2 -time).